edge-modules/ModuleLib/ResultType.cs

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.ModuleUtil
+{
+    public enum ResultType
```

Comment:
Please rename it as "TestResultType" or "TestOperationResultType"

Reply:
done